src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -93,7 +95,7 @@ public PersistentDataStore() {
     List<Conversation> conversations = new ArrayList<>();
 
     // Retrieve all conversations from the datastore.
-    Query query = new Query("chat-conversations").addSort("creation_time", SortDirection.ASCENDING);
```

Comment:
Why was this create_time before and why is ok to change it to message_count?

Reply:
As far as I know is just the order in which the queue of Entities, then in this case I use it for my convenience, because also there isn't any method that took advantage of this sorting. Then I decided to change.

Reply:
But actually… now that I giving a second thought about it, the only thing it may happen is that the conversations listed in the conversation page will be also sorted.

Reply:
Yea. I think this may have something to do with how the messages are presented to the user in chats. If thats the case we may not want to change it.

Reply:
It's not about the messages it's about how it's presented the topics of conversations. The messages in a conversation will still appear in chronological order.

Reply:
Lol alright. At this point you have a better understanding of this than me. Maybe ill take a closer look some other time.